jupyterhub/notebook-images/overlays/cuda/cuda-10-1-ubi7-build-chain.yaml

```diff
@@ -0,0 +1,258 @@
+apiVersion: v1
+items:
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    labels:
+      appName: 10.1-cuda-chain-ubi7
+      appTypes: cuda-build-chain
+    name: cuda
+  spec:
+    lookupPolicy:
+      local: true
+- apiVersion: image.openshift.io/v1
+  kind: ImageStream
+  metadata:
+    labels:
+      appName: 10.1-cuda-chain-ubi7
+      appTypes: cuda-build-chain
+    name: python-36-ubi7
+  spec:
+    lookupPolicy:
+      local: true
+- apiVersion: build.openshift.io/v1
+  kind: BuildConfig
+  metadata:
+    labels:
+      appName: 10.1-cuda-chain-ubi7
+      appName1: 10.1-base-ubi7
+      appTypes: cuda-build-chain
+    name: 10.1-base-ubi7
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: cuda:10.1-base-ubi7
+    resources:
+      limits:
+        cpu: "4"
+        memory: 4Gi
+      requests:
+        cpu: "4"
```

Comment:
4 cpu / 4Gi can be a lot in a smaller (CRC) cluster.  I'm not sure if CRC has gpu support, so that overlay can probably be skipped on crc runs, but maybe a smaller request, just in case it could work.  If you like these values a lot though, I won't object any further.

Reply:
Yeah, it is a lot, but with anything below that the builds can fail. I believe CRC can potentially leverage GPU (at least someone was trying to get it working), but at that point it is very special case, so we could assume a user would also provide more resources

Reply:
Ok.  Maybe just be sure to include something in the docs about disabling the cuda overlay (or enabling the cuda overlay if maybe it shouldn't be on by default).

Reply:
Yeah, it should not be on by default. I'll cover that in the docs